packages/common/src/search/_internal/hubEventsHelpers/processFilters.ts

```diff
@@ -31,20 +46,43 @@ export function processFilters(filters: IFilter[]): Partial<GetEventsParams> {
   if (tags?.length) {
     processedFilters.tags = tags;
   }
+  const readGroupIds = getOptionalPredicateStringsByKey(
```

Comment:
Similar point, make the IQuery filter name `readGroupId` and update the `hubSearch` docs to reflect this. 

```ts
const readGroupIds = getOptionalPredicateStringsByKey(filters, 'readGroupId');
if (readGroupIds?.length) {
  processedFilters.readGroups = readGroupIds;
}
```